### PR TITLE
Allow user to specify a partial gpu name to the forceAdaptor commandline argument

### PR DESCRIPTION
**Pull request by rgba16f**

### CODE REVIEW

**Review by amzn-tommy — APPROVED** [review]

File: Gems/Atom/RPI/Code/Source/RPI.Private/RPISystemComponent.cpp [hunk 1]
```diff
@@ -56,6 +56,7 @@ namespace AZ
         void RPISystemComponent::GetRequiredServices(ComponentDescriptor::DependencyArrayType& required)
         {
             required.push_back(RHI::Factory::GetComponentService());
+            required.push_back(AZ_CRC_CE("LoggerService"));
```
Comment (moudgils) [inline Gems/Atom/RPI/Code/Source/RPI.Private/RPISystemComponent.cpp:59]: Is this intentional?

**Review by moudgils — APPROVED** [review]
Other than the one comment(Remove if un-intentional) looks good.

**Comment by jeremyong-az** [comment]
Excellent QoL improvement

File: Gems/Atom/RPI/Code/Source/RPI.Private/RPISystemComponent.cpp (hunk 1 above)
Comment (rgba16f) [reply Gems/Atom/RPI/Code/Source/RPI.Private/RPISystemComponent.cpp]: I removed it.

**Review by AMZN-tpeng — APPROVED** [review]

**Pull request merged**